clang/lib/Serialization/ASTWriterStmt.cpp

```diff
@@ -466,6 +466,12 @@ void ASTStmtWriter::VisitCoyieldExpr(CoyieldExpr *E) {
   Code = serialization::EXPR_COYIELD;
 }
 
+void ASTStmtWriter::VisitCXXReflectExpr(CXXReflectExpr *E) {
+  VisitExpr(E);
+  Record.AddSourceLocation(E->getOperatorLoc());
+  Code = serialization::EXPR_REFLECT;
+}
```

Comment:
Given it is not implemented in reader side and this is not tested, I think it will be better to list it as not implemented directly.